.claude/hooks/post_edit.py

```
@@ -0,0 +1,26 @@
+import subprocess
+
+data = sys.stdin.buffer.read()
+
+# otogidb/.claude/hooks/ -> otogidb/.claude/ -> otogidb/ -> OtogiReverse/
+here = os.path.dirname(os.path.abspath(__file__))
+parent_hook = os.path.normpath(
+    os.path.join(here, "..", "..", "..", ".claude", "hooks", "post_edit.py")
+)
+
+if os.path.isfile(parent_hook):
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)